### PR TITLE
feat: adding RSS and JSON feeds to the website, using `@nuxtjs/feed`

### PR DESCRIPTION
**Pull request by landjonathan**
Useful for getting updates whenever a module is added. 
This is a fairly straightforward implementation of `@nuxtjs/feed`, including most of the module info.

### CODE REVIEW

File: nuxt.config.ts [hunk 1]
```diff
@@ -40,5 +41,70 @@ export default /*<NuxtConfig>*/ {
...
+        link: 'https://modules.nuxtjs.org/',
+        description: 'Discover our list of modules to supercharge your Nuxt project. Created by the Nuxt team and community.',
+      }
+
+      const modules = await $content()
+        .sortBy('createdAt', 'desc')
+        .limit(10)
+        .fetch()
+
+      for (const { name, website, createdAt, description, icon, category, type, maintainers } of modules) {
```
Comment (pi0) [inline nuxt.config.ts:64]: I asume `createdAt` is based on `fs.stat` for module meta, which should be fine. /cc @Atinux @benjamincanac

**Review by pi0 — APPROVED** [review]
Thanks!

File: nuxt.config.ts (hunk 1 above)
Comment (landjonathan) [reply nuxt.config.ts:64]: Not sure this answers your question, but `createdAt` is supplied by `$content`. From what I see this project uses the default implementation.

Comment (Atinux) [reply nuxt.config.ts:64]: It should be good normally, it is based from `fs.stat`, I may be afraid that it won't respect the filesystem date when deploying on Vercel (https://github.com/nuxt/content/blob/dev/packages/content/lib/database.js#L243)

Comment (pi0) [reply nuxt.config.ts:64]: Hmm that's definitely an issue with vercel then :( Maybe we can utilize https://github.com/MestreLion/git-tools/blob/master/git-restore-mtime

Comment (landjonathan) [reply nuxt.config.ts:64]: Just curious, is this a general problem with the content module + Vercel? Or is it just an issue for this project?

Comment (Atinux) [reply nuxt.config.ts:64]: I think this should be an issue in most providers since they actually don't really clone the repo but download the tar.

Comment (Atinux) [reply nuxt.config.ts:64]: Looks like it is the same date: https://modules-3ergmki3y.vercel.app/rss.xml

**Comment by Atinux** [comment]
Since we cannot have the dateAdded for each module, is this still relevant?

**Comment by landjonathan** [comment]
Would it be possible to use the build time, and then cache it? So that only yml files without a cached timestamp will get a new one on during build? Can the Nuxt cache do something like that?

**Comment by pi0** [comment]
@landjonathan Would be nice idea doing in [modules.ts](https://github.com/nuxt/modules/blob/master/scripts/modules.ts)

You can pick both `time.updated` and `time.modified` fields by making a http request to npm registry. [example](http://registry.npmjs.org/@nuxt/loading-screen)

**Comment by pi0** [comment]
Hi @landjonathan. Sorry that this PR got stalled. We made several major changes to website structure and now both runtime checking and last updated at is available in runtime. Moving this idea to https://github.com/nuxt/modules/issues/252

**Pull request closed without merging**